### ISSUE TITLE
OS app takes 2-3 minutes to launch on mac

### ISSUE THREAD

**Issue by DavidGoldwasser** [NREL/OpenStudio#3235]
2.5.0 doesn't seem to be affect by this, but later installers including 2.6.0 and 2.6.1 are. Processor is at 100% over this time.

**Comment by jmarrec** [comment]
Just tried it: takes a couple of seconds on my mac with 2.6.0.

macOS High Sierra, 10.13.6 (17G65)

**Comment by DavidGoldwasser** [comment]
Larry was seeing this as well. I did go off network, turn off wifi, and setup an empty directory as measures dir, still took a long time.

**Comment by jmarrec** [comment]
Is that just launching the os app or double clicking an osm file to open it?

**Comment by DavidGoldwasser** [comment]
Just launching the app.

**Comment by DavidGoldwasser** [comment]
When I installed pre 2.7.0 it took a long time then I got error about default libraries, which I had not ever changed on purpose. When I restored default libraries and then re-launched app, it just took a few seconds. Went back to 2.6.0 and it also launches quickly now.

@ljbrackney if you are still seeing slow launch try to reset the libraries. Assuming that works for you I think we can close this.
